libpod/image/image.go

```diff
@@ -166,6 +165,14 @@ func (i *Image) reloadImage() error {
 	return nil
 }
 
+// stringSha256 strips sha256 from user input
+func stripSha256(name string) string {
+	if strings.HasPrefix(name, "sha256:") {
+		return name[7:]
```

Comment:
Do you need a CYA len check on name here?  If name is less than 7, life could be unhappy.

Reply:
If it has prefix SHA256: it's definitely at least 7 characters... Though it might be worth it to add a check that the string is at least 8 characters, because this would allow us to use "sha256:" with no actual image ID, and we should probably check that they provided at least one hex character.

Reply:
fixered